kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue

```diff
@@ -40,7 +40,7 @@
       <KGridItem>{{ $tr('doNotSeeYourFacility') }}</KGridItem>
       <KGridItem>
         <KButton
-          :text="coreString('addNewAddressAction')"
+          :text="getCommonSyncString('addNewAddressAction')"
```

Comment:
Shoot, I should have checked elsewhere-- I noticed this in the `SelectDeviceForm`